### ISSUE TITLE
Inference Code and Model Weights

### ISSUE THREAD

**Issue by vishxlyadav** [PhyRecon/PhyRecon#6]
Thanks for the great work.
Would you be releasing the pre-trained model weights?
Also, how can I get inference on my own data?

**Comment by DaLi-Jack** [comment]
Hi! Thank you for your interest in our work!
I am sorry I didn't save the checkpoint for each scene. However, you can find the reconstructed results [here](https://drive.google.com/file/d/19nFdVX9wcPKXyhFHlPwgvtjYPTrDDzyR/view).
We use the same data format as MonoSDF, so you can refer to [their data processing code](https://github.com/autonomousvision/monosdf#custom-dataset) for more details. Additionally, we use the ground truth instance mask annotations from each dataset. If your data lacks multi-view consistency instance mask annotations, you can use [SAM2](https://github.com/facebookresearch/sam2) to generate them.
 Hope this helps!

**Comment by vishxlyadav** [comment]
So I understand that this method overfits on a single scene (similar to that of NeRFs) and you need checkpoints corresponding to a single scene to reconstruct it.
Can you confirm that?

**Comment by DaLi-Jack** [comment]
Yes, you’re right. Our method is to overfit a single scene. For each scene, we need to retrain our model.

**Comment by vishxlyadav** [comment]
Thanks a lot.
